unified-prototype/unified-plugin/plugin-android/src/main/java/org/gradle/api/experimental/android/AndroidDSLSupport.java

```diff
@@ -27,4 +30,8 @@ public static <T> void ifPresent(Property<T> property, Action<T> action) {
             action.execute(property.get());
         }
     }
+
+    public static VersionCatalog getLibs(Project project) {
```

Comment:
What are we using this for?

Reply:
Nothing presently, I was trying some things and missed deleting it.  I'll remove.